app/src/components/fields/fieldErrorHint/fieldErrorHint.jsx

```diff
@@ -25,17 +25,17 @@ const cx = classNames.bind(styles);
 const messages = defineMessages({
   loginHint: {
     id: 'RegistrationForm.loginHint',
-    defaultMessage:
-      'Login should have size from 1 to 128 symbols, latin, numeric characters, hyphen, underscore, dot.',
+    defaultMessage: 'User Name should contain only valid symbols and at least 1 Latin symbols',
   },
   nameHint: {
     id: 'RegistrationForm.nameHint',
     defaultMessage:
-      'Full name should have size from 3 to 256 symbols, latin, cyrillic, numeric characters, hyphen, underscore, dot, space.',
+      'Full Name should contain only valid symbols and at least 3 Latin or Cyrillic symbols',
   },
   passwordHint: {
     id: 'RegistrationForm.passwordHint',
-    defaultMessage: 'Password should have size from 4 to 256 symbols.',
+    defaultMessage:
+      'Password should contain at least 4 characters; a special symbol; upper-case (A - Z); lower-case',
```

Comment:
There is not such restriction in the system, now user can create password from 4 to 256 symbols only even with cyrillic symbols.
Please discuss it with design team and note that password requirements didn't change.